### PR TITLE
06 react state part 2

### PR DESCRIPTION
**Pull request by davelosert**
@goloroden : And here is the Code for the next episode.

### CODE REVIEW

File: 06-user-events/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "06-user-events",
+  "version": "1.0.0",
+  "description": "Vorstellung von React-State zum Dynamisieren einer React-Applikation",
```
Comment (goloroden) [inline 06-user-events/package.json:4]: This is the old description of episode 5, I guess, can you please update this one?

File: 06-user-events/src/app/CounterStateless.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export class CounterStateless extends React.PureComponent {
+  handleClick () {
+    this.props.incrementCounter();
+  }
+
+  render () {
+    return (
+      <div>
+        <h3>Counter Pure is: {this.props.counterValue}</h3>
```
Comment (goloroden) [inline 06-user-events/src/app/CounterStateless.jsx:11]: Shouldn't this be `Counter Stateless`?

File: 06-user-events/src/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Einführung in React-State</title>
```
Comment (goloroden) [inline 06-user-events/src/index.html:5]: This, too, seems to be the old title.

**Comment by davelosert** [comment]
@goloroden : Should have fixed everything from your comments - you want to have another look?

**Pull request merged**